### PR TITLE
GitHub actions

### LINKED ISSUE #28: Switch from Travis CI to Github Actions
https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing

### PR DESCRIPTION
**Pull request by DJMcNab**
Closes #28

### CODE REVIEW

**Comment by DJMcNab** [comment]
Not sure why the actions don't run on this commit, but you should be able to see that they pass on the head branch

**Comment by bjorn3** [comment]
Github actions is activated once a workflow is pushed to the default branch.

File: README.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ to Miri and fix it there.
 
 Licensed under either of
 
-* Apache License, Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE)
-* MIT license ([LICENSE-MIT](LICENSE-MIT)
+- Apache License, Version 2.0 ([LICENSE-APACHE](LICENSE-APACHE)
+- MIT license ([LICENSE-MIT](LICENSE-MIT)
```
Comment (bjorn3) [inline README.md]: Why did you change the bullet style?

Comment (DJMcNab) [reply README.md]: Both because `prettier` has that as the preferred style, and because stackoverflow told me that GitHub actions needs an additional commit after the workflow file is created. It turned out my issue was with my yaml formatting, which for some reason invalid syntax on the second item indented at the same level.

Comment (bjorn3) [reply README.md]: Can you move this to a separate PR?

**Review by bjorn3 — APPROVED** [review]
Thanks! Maybe caching could be added in a next PR? 2m 47s already is not bad though.

**Comment by bjorn3** [comment]
@oli-obk Maybe you want to disable the Travis CI integration? It isn't strictly necessary though.

**Comment by DJMcNab** [comment]
We don't do any building at the moment, so I think caching would probably be a net loss.
